Makefile

```diff
@@ -255,8 +255,21 @@ deploy-aws-account-operator:
 # Create zero size account pool
 	@oc apply -f hack/files/aws_v1alpha1_zero_size_accountpool.yaml
 
-.PHONY: deploy
-deploy: check-aws-credentials deploy-aws-account-operator-credentials deploy-aws-account-operator
+.PHONY: deploy-local
+deploy-local: FORCE_DEV_MODE?=local
+deploy-local: check-aws-credentials deploy-aws-account-operator-credentials deploy-aws-account-operator
+	@operator-sdk up local --namespace=$(OPERATOR_NAMESPACE)
+
+.PHONY: deploy-cluster
+deploy-cluster: FORCE_DEV_MODE?=cluster
+deploy-cluster: isclean
```

Comment:
Oops, this needs to pull in the same things as L260

Reply:
Actually ended up doing it the other way: neither `deploy-*` target pulls in the predeploy steps.